plugin-api/src/main/java/org/hyperledger/besu/plugin/services/trielogs/TrieLogEvent.java

```diff
@@ -14,12 +14,17 @@
  */
 package org.hyperledger.besu.plugin.services.trielogs;
 
+import com.google.common.annotations.VisibleForTesting;
+
 /** A TrieLog event. */
 public interface TrieLogEvent {
   /** The type of the event. */
   enum Type {
     /** TrieLog added event type */
-    ADDED
+    ADDED,
+    /** dummy value (this comment required by javadoc) * */
+    @VisibleForTesting
+    UNKNOWN
```

Comment:
Would prefer to not expose this on the public API. Is there another way of testing that it only works for the ADDED type?

Reply:
The options I considered were:
1. This
2. Subclass TrieLogEvent for the test, but is essentially duplicating this part of the code so not really a good test.
3. No test for this case.

Reply:
To be honest, I'm not sure why there is both a subclass representing the event type as well this enum...maybe don't need the enum at all?

Reply:
Maybe no test is ok. Right now it's not possible to create a TrieLogEvent with anything other than an ADDED event

Reply:
removed the test